src/components/media-link/media-link.stories.mdx

```diff
@@ -0,0 +1,82 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { mediaArgTypes } from '../../objects/media/demo/arg-types.js';
+import { avatarArgTypes } from '../avatar/demo/arg-types.js';
```

Comment:
Because this component extends the Media object and passes along a lot of properties to Avatar, I decided to put those patterns' `argTypes` into separate files so I could import them into this one and avoid writing the same controls over and over.